### ISSUE TITLE
Icons do not load

### ISSUE THREAD

**Issue by kalypzo** [sherweb/ng2-materialize#263]
## Expected Behavior
When applying tag `<i mz-icon class="col s12 m2" [icon]="'cloud'"></i>`, an icon should show

## Current Behavior
No icon shows.

## Possible Solution
Make sure to include the following in your index.html:
`<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">`
There was no mention of this in the initial setup documentation. I couldn't figure out why I couldn't get the icons to work. Finally came across a post somewhere else that mentioned this. Posting this in case someone runs into the same issue I did.

**Comment by LiadIdan** [comment]
@kalypzo it is mentioned in the README file under the installation section and marked as optional:
```
npm install --save ng2-materialize
npm install --save mdi    #optional
```
#### .angular-cli.json
```diff
"styles": [
  "styles.scss",
+ "../node_modules/materialize-css/dist/css/materialize.min.css",
+ "../node_modules/mdi/css/materialdesignicons.min.css"   #optional
]
```

**Comment by jfcere** [comment]
@LiadIdan the mention is for [Material Design Icon](https://materialdesignicons.com/) (community project based on Google Material Icons) which should be used with `mz-icon-mdi` directive but I believe @kalypzo pointed out that we have no mention how to include [Material Icons](https://material.io/icons/) (Google Material Icons) when using `mz-icon` directive.

**Comment by LiadIdan** [comment]
@jfcere I just realized that it's two separate packages :sweat_smile:

**Comment by jfcere** [comment]
@LiadIdan don't worry ... it is not that obvious :)

**Comment by kalypzo** [comment]
I just realized I shouldn't need to use the link href. However, I did follow the instructions with npm install and the angular-cli.json, it just won't work. However, it does work in the sample project, but it's using an older version of the angular-cli. Maybe I need to downgrade? Everything in my package.json, angular-cli.json, tsconfig.json, and tsconfig.app.json matches. Will troubleshoot further and let you know my findings.
